modules/home/assets/js/components/addons-section.js

```diff
@@ -0,0 +1,51 @@
+import { Box, Paper, Typography } from '@elementor/ui';
+import List from '@elementor/ui/List';
+import Link from '@elementor/ui/Link';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardActions from '@elementor/ui/CardActions';
+import CardContent from '@elementor/ui/CardContent';
+import CardMedia from '@elementor/ui/CardMedia';
+
+const Addons = ( { ...props } ) => {
+	const addonsArray = props.addonsData.repeater.filter( ( item ) => false === item.is_installed );
```

Comment:
Not every item has the `is_installed` property.

```suggestion
	const addonsArray = props.addonsData.repeater.filter( ( item ) => true !== item?.is_installed );
```